<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDrawBoxWidget" name="paintArea" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>600</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>320</height>
     </size>
    </property>
   </widget>
   <widget class="QLedIndicator" name="recordButton" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wavfile status:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="audioDeviceLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenWavfile"/>
    <addaction name="actionSaveHistogram"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Configure">
    <property name="title">
     <string>&amp;Configure</string>
    </property>
    <addaction name="actionConfigFilter"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuA_bout">
    <property name="title">
     <string>A&amp;bout</string>
    </property>
    <addaction name="actionAboutThis"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Configure"/>
   <addaction name="menuA_bout"/>
  </widget>
  <action name="actionSaveHistogram">
   <property name="text">
    <string>Save &amp;Histogram</string>
   </property>
  </action>
  <action name="actionAudioSetting">
   <property name="text">
    <string>&amp;Setting</string>
   </property>
  </action>
  <action name="actionConfigFilter">
   <property name="text">
    <string>&amp;Filter</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAboutThis">
   <property name="text">
    <string>About &amp;This</string>
   </property>
  </action>
  <action name="actionOpenWavfile">
   <property name="text">
    <string>Open &amp;Wavfile</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDrawBoxWidget</class>
   <extends>QWidget</extends>
   <header>qdrawboxwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLedIndicator</class>
   <extends>QWidget</extends>
   <header>qledindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
